js/index.js

```diff
@@ -0,0 +1,169 @@
+let bookCounter;
+const sectionBooks = document.querySelector('.bookshelf');
+const sectionAddBook = document.querySelector('.addBooks');
+const inputTitle = document.getElementById('title');
+const inputAuthor = document.getElementById('author');
+const inputPages = document.getElementById('pages');
+const inputreadYes = document.getElementById('readYes');
+const inputreadNo = document.getElementById('readNo');
+const buttonSubmit = document.getElementById('buttonSubmitBook');
+const warning = document.getElementById('warning');
+const buttonAddBook = document.getElementById('buttonAddBook');
+
+let myLibrary = [];
+let myLibraryStringfied = [];
+
+if (localStorage.getItem('myLibrary')) {
+  myLibraryStringfied = localStorage.getItem('myLibrary').split('/');
+  myLibrary = myLibraryStringfied.map((book) => JSON.parse(book));
+  // eslint-disable-next-line no-use-before-define
+  myLibrary.map((book) => displayBook(book));
+  let counter = 0;
+  for (let i = 0; i < myLibrary.length; i += 1) {
+    if (myLibrary[i].id > counter) counter = myLibrary[i].id;
+  }
+  bookCounter = counter;
+} else {
+  localStorage.setItem('myLibrary', '');
+  bookCounter = 0;
+}
+
+sectionAddBook.style.display = 'none';
+
+buttonAddBook.addEventListener('click', (event) => {
+  if (sectionAddBook.style.display === 'none') {
+    sectionAddBook.style.display = 'block';
+    event.currentTarget.textContent = 'Close';
+  } else {
+    sectionAddBook.style.display = 'none';
+    event.currentTarget.textContent = 'New Book';
+  }
+});
+
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages > 0 ? pages : 'invalid number of pages';
+  this.read = read;
+
+  this.info = function () {
+    // eslint-disable-next-line no-nested-ternary
+    return (`${this.title} by ${this.author}, ${this.pages}${typeof this.pages === 'number' ? this.pages === 1 ? ' page' : ' pages' : ''}, ${this.read ? 'read' : 'not read yet'}`);
+  };
+  this.toggleread = function () {
+    this.read = !(this.read);
+  };
+}
+
+function verifyInputs() {
+  if (
+    !(inputAuthor.value)
+        || !(inputPages.value)
+        || !(inputTitle.value)) return false;
+  if (!(inputreadYes.checked) && !(inputreadNo.checked)) return false;
+  return true;
+}
+
+function clearInputs() {
+  inputTitle.value = '';
+  inputAuthor.value = '';
+  inputPages.value = '';
+  warning.textContent = '';
+  if (inputreadYes.checked) inputreadYes.checked = false;
+  else inputreadNo.checked = false;
+}
+
+function displayBook(book) {
```

Comment:
- [ ] <b>OPTIONAL:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```